### PR TITLE
Q2 Commerce & Open Source release (2.3.2, 2.2.9, 2.1.18)

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) publishes all Q2-related release documentation.

whatsnew
Published all documentation updates related to the Q2 Commerce and Open Source release.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4825) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4825) before we can accept your contribution.<br/>**7** out of **8** committers have signed the CLA.<br/><br/>:white_check_mark: mage2pratik<br/>:white_check_mark: keharper<br/>:white_check_mark: mageprince<br/>:white_check_mark: yogeshsuhagiya<br/>:white_check_mark: jeff-matthews<br/>:white_check_mark: rogyar<br/>:white_check_mark: ParthBrahmbhatt<br/>:x: AlexMaxHorkun

**Comment by jeff-matthews** [comment]
running tests

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (dshevtsov) [inline .vscode/settings.json]: remove this file

File: guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
```
Comment (dshevtsov) [inline guides/v2.1/release-notes/ReleaseNotes2.1.18CE.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.1/release-notes/ReleaseNotes2.1.18EE.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
```
Comment (dshevtsov) [inline guides/v2.1/release-notes/ReleaseNotes2.1.18EE.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.2/release-notes/ReleaseNotes2.2.9CE.md [hunk 4]
```diff
@@ -0,0 +1,795 @@
+---
```
Comment (dshevtsov) [inline guides/v2.2/release-notes/ReleaseNotes2.2.9CE.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.2/release-notes/ReleaseNotes2.2.9EE.md [hunk 5]
```diff
@@ -0,0 +1,846 @@
+---
```
Comment (dshevtsov) [inline guides/v2.2/release-notes/ReleaseNotes2.2.9EE.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md [hunk 6]
```diff
@@ -36,9 +36,9 @@ Follow these steps to download and apply this patch:
```
Comment (dshevtsov) [inline guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.3/release-notes/ReleaseNotes2.3.0OpenSource.md [hunk 7]
```diff
@@ -35,9 +35,9 @@ Follow these steps to download and apply this patch:
```
Comment (dshevtsov) [inline guides/v2.3/release-notes/ReleaseNotes2.3.0OpenSource.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md [hunk 8]
```diff
@@ -75,7 +75,7 @@ The Admin order creation workflow has been refactored to eliminate delays when e
```
Comment (dshevtsov) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1Commerce.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.md [hunk 9]
```diff
@@ -62,7 +62,7 @@ The Admin order creation workflow has been refactored to eliminate delays when e
```
Comment (dshevtsov) [inline guides/v2.3/release-notes/ReleaseNotes2.3.1OpenSource.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md [hunk 10]
```diff
@@ -0,0 +1,1159 @@
+---
```
Comment (dshevtsov) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md]: the file name doesn't follow the DevDocs naming convention

File: guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md [hunk 11]
```diff
@@ -0,0 +1,1047 @@
+---
```
Comment (dshevtsov) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.md]: the file name doesn't follow the DevDocs naming convention

**Comment by dshevtsov** [comment]
Comments to `whatsnew`:
1. Is _Q2 Commerce release_ a recognizable term by the Community?
2. Does this include Open Source release updates?

**Comment by jeff-matthews** [comment]
Updated the PR title to include Open Source

File: .vscode/settings.json (hunk 1 above)
Comment (jeff-matthews) [reply .vscode/settings.json]: done in https://github.com/magento/devdocs/pull/4827

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**